letsencrypt/display/util.py

```diff
@@ -520,6 +555,23 @@ def checklist(self, message, tags, default=None, cli_flag=None, **kwargs):
         else:
             return OK, default
 
+    def directory_select(self, message, default=None, cli_flag=None):
+        """Simulate prompting the user for a directory.
+
+        This function returns default if it is not ``None``, otherwise,
+        an exception is raised.
+
+        :param str message: prompt to give the user
+        :param default: default value to return (if one exists)
+        :param str cli_flag: option used to set this value with the CLI
```

Comment:
I didn't understand how `cli_flag` is used (although it already exists in the code elsewhere). Is it possible to give a slightly more detailed explanation in the docstring?